examples/accessibility.html

```diff
@@ -0,0 +1,66 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>&lt;model-viewer&gt; - Examples</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, user-scalable=no,
+  minimum-scale=1.0, maximum-scale=1.0">
+  <link type="text/css" href="styles/examples.css" rel="stylesheet" />
+
+  <!-- The following libraries and polyfills are recommended to maximize browser support -->
+  <!-- Include prismatic.js for Magic Leap support: -->
+  <!--<script src="../node_modules/@magicleap/prismatic/prismatic.min.js"></script>-->
+
+  <!-- Include the Custom Elements ES5 adapter shim if you are loading an IE11-compatible bundle -->
+  <!--<script src="../node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js"></script>-->
+
+  <!-- Web Components polyfill is required to support Edge and Firefox < 63: -->
+  <script src="../node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+
+  <!-- Intersection Observer polyfill is required for Safari and IE11 -->
+  <script src="../node_modules/intersection-observer/intersection-observer.js"></script>
+
+  <!-- Resize Observer polyfill is optional, and improves resize behavior in non-Chrome browsers: -->
+  <script src="../node_modules/resize-observer-polyfill/dist/ResizeObserver.js"></script>
+
+  <!-- Fullscreen polyfill is required for using experimental AR features in Canary: -->
+  <script src="../node_modules/fullscreen-polyfill/dist/fullscreen.polyfill.js"></script>
+</head>
+<body>
+    <div id="container">
+        <h1>&lt;model-viewer&gt; - Accessibility Tests</h1>
+        <div class="desc">
+            <p>There are several different tests in order to investigate different aspects of accessibility and <code>&lt;model-viewer&gt;</code>. This page enumerates the test cases, and provides links to the individual scenarios.</p>
+            <p>Note that all these examples include <code>alt</code>.</p>
+        </div>
+        <ul class="desc">
+            <li><a href="default.html">default.html</a>: The barebones default case.</li>
+            <li><a href="accessibility/poster.html">accessibility/poster.html</a>: A single sample exercising <code>poster</code>.</li>
+	          <li><a href="accessibility/reveal.html">accessibility/reveal.html</a>: A single sample exercising <code>poster</code>, <code>preload</code>, and <code>reveal-when-loaded</code>.</li>
+	          <li><a href="accessibility/controls.html">accessibility/controls.html</a>: A single sample exercising <code>controls</code>.</li>
+	          <li><a href="accessibility/auto-rotate.html">accessibility/auto-rotate.html</a>: A single sample exercising <code>auto-rotate</code>.</li>
+	          <li><a href="accessibility/multiple.html">accessibility/multiple.html</a>: Multiple models on the same page with a variety of attributes.</li>
```

Comment:
Is there a version of this where all examples live on one page? I'm not sure what we get out of splitting the examples up into one test case per page - did you have anything in mind?

Reply:
The `multiple.html` does cover most of the scenarios. I thought it might make tests and conversations about the pages simpler to exercise a single scenario on an individual page, especially since the "many models on a page" was a specific scenario that we wanted to test.

Happy to change it, though!